recipes/bamcmp/meta.yaml

```
@@ -7,7 +7,9 @@ source:
   md5: 05e583af007f86b275bcf95eceea7ba7
 
 build:
-  number: 4
+  number: 5
+  run_exports:
+    - {{ pin_subpackage("bamcmp", max_pin="x.x.x") }}  
```

It is very hard for me to decide what `max_pin` value to use too because many of the packages do not specify their version policy, but according to current version (2.2) it should be:
```suggestion
    - {{ pin_subpackage("bamcmp", max_pin="x") }}  
```

```
semantic versioning 	{{ pin_subpackage("myrecipe", max_pin="x") }}
```

---

agreed, we'll go with suggestion

@sunqian2116 reverse if you think otherwise